### DISCUSSION TITLE
Multiply/Add constant value in custom sensors/curves

### DISCUSSION THREAD

**Discussion by Migratingcocofruit** [Rem0o/FanControl.Releases#981] [Ideas]
This could be used to create weighted averages and offsets. 
This could be useful for situations where certain components should be kept at a lower temp from others, such as RAM, or when you don't mind it if some component gets a bit toastier than others.

**Comment by TrickyWidget** [comment]
I second this idea.  I'd like to create a sensor that tracks how close my GPU is to throttle temp.  The card doesn't provide that value, but I could calculate it if I could make a Mix sensor that subtracts the current temp from an arbitrary value (the throttle temp).

**Reply by Rem0o** [reply]
As a workarround, you can create a "file" sensor, write 90 in it, save, and you have your throttle temp to use in a mix
